packages/constants/src/constants.ts

```diff
@@ -133,6 +133,11 @@ export const MC_API_PROXY_TARGETS = {
 } as const;
 export type TMcApiProxyTargets = typeof MC_API_PROXY_TARGETS[keyof typeof MC_API_PROXY_TARGETS];
 
+export const MC_API_SUPPORTED_HEADERS = {
+  TOKEN_RETRY: 'X-Force-Token',
```

Comment:
We might have more but not a concern here I think.

Reply:
Hmm I meant to use this constants object: https://github.com/commercetools/merchant-center-application-kit/blob/ebb2441cfb5ee126a1be0ea0add017ec53aefbba/packages/application-shell/src/constants.ts#L1-L10

Reply:
Ah dang it. Didn't know this one. Odd it's defined in app-shell even though we have a shared constants file.